airflow/kubernetes/worker_configuration.py

```diff
@@ -127,6 +127,10 @@ def _get_init_containers(self) -> List[k8s.V1Container]:
                     name='GIT_SSH_KEY_FILE',
                     value='/etc/git-secret/ssh'
                 ),
+                k8s.V1EnvVar(
+                    name='GIT_SYNC_ADD_USER',
+                    value='true'
+                ),
```

Comment:
using the same config on the kubernetes worker as well, without this,  git sync with ssh may not work